src/Hazelcast.Net/HazelcastOptions.ClusterOptions.cs

```diff
@@ -39,10 +39,10 @@ public partial class HazelcastOptions : IClusterOptions // ClusterName, Subscrib
         internal const string DefaultClusterName = "dev";
 
         /// <summary>
-        /// Gets the default client name prefix, which is <c>"hz.client_"</c>.
+        /// Gets the default client name prefix, which is <c>"hznet.client_"</c>.
```

Comment:
I don't get the benefits of doing things differently from our reference implementation, Java client. If there is such a benefit, we need to discuss as a group to decide whether we should not follow the reference.

Reply:
But then again, "hz.client_" is for Java, but other clients _already_ use a different name, so I aligned with them...

Reply:
I am pretty sure I have seen another client use a different name on MC, but everyone seems to agree that every client use the `hz.client_<n>` pattern. I am pretty sure that any value could be used here, since this is only used to create a client name when the user does not specify one, but the user could really specify anything. Anyways. Reverting the change.